src/vscode-bicep/src/test/e2e/pasteAsBicep.test.ts

```diff
@@ -135,7 +135,7 @@ resource aksCluster1 'Microsoft.ContainerService/managedClusters@2021-05-01' = {
     });
     await vscode.window.showTextDocument(textDocument);
 
-    vscode.env.clipboard.writeText(json);
+    await vscode.env.clipboard.writeText(json);
```

Comment:
Hopefully this is the cause of #9703 (and proves why I wanted to enable this rule)